framework/src/modules/interoperability/utils.ts

```diff
@@ -40,3 +40,27 @@ export const getCCMSize = (ccm: CCMsg) => {
 
 	return serializedCCM.byteLength;
 };
+
+export const getEncodedSidechainTerminatedCCMParam = (
```

Comment:
Optional: For this PR I see only one instance of this util function being used so maybe we don't need a separate util for it